apps/allocations/app/components/App/App.js

```diff
@@ -2,21 +2,50 @@ import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 
 import { useAragonApi } from '../../api-react'
-import { Button, Header, IconPlus, Main, SidePanel } from '@aragon/ui'
+import { Button, Header, IconPlus, Main, Modal, SidePanel } from '@aragon/ui'
 
 import { IdentityProvider } from '../../../../../shared/identity'
 import { Empty } from '../Card'
 import { NewAllocation, NewBudget } from '../Panel'
-import { Accounts, Payouts } from '.'
+import { AllocationsHistory, Budgets } from '.'
 
 const ASSETS_URL = './aragon-ui'
 
 const nameSorter = (a, b) => a.data.name.toUpperCase() > b.data.name.toUpperCase() ? 1 : -1
 
 const App = () => {
   const [ panel, setPanel ] = useState(null)
+  const [ modal, setModal ] = useState({ visible: false, budgetId: null })
   const { api, appState } = useAragonApi()
-  const { accounts = [], balances = [], payouts = [] } = appState
+  const {
+    // backend stub, remove
+    budgets = [
+      {
+        budgetId: '0',
+        data: {
+          name: 'Marketing',
+          amount: String(80000.123856789012345678e18),
+          currency: 'ETH',
+          allocated: String(23000e18),
+          inactive: false,
+        }
+      },
+      {
+        budgetId: '1',
+        data: {
+          name: 'Hacktivism',
+          amount: String(38000.123856789012345678e18),
+          currency: 'DAI',
+          allocated: String(37200e18),
+          inactive: true,
+        }
+      },
+    ],
+    balances = [],
+    entries = [],
```

Comment:
We are we re-adding entries? this was previously removed since it is part of the former address book integration, that now is becoming a dao wide IdentityProvider.

Please remove this regression.